src/components/Button/index.tsx

```diff
@@ -204,7 +204,7 @@ const getStyledButton = (
 	return ColouredButton;
 };
 
-const Base = (props: ThemedButtonProps) => {
+const Base = React.forwardRef((props: ThemedButtonProps, ref: any) => {
```

Comment:
Let's type the `ref`

Reply:
@karaxuna any reason why you marked this as resolved? I still see the `ref` typed as `any`.